bridges/centralized-ethereum/src/actors/dr_sender.rs

```diff
@@ -153,13 +157,26 @@ impl DrSender {
     }
 }
 
-fn deserialize_and_validate_dr_bytes(dr_bytes: &[u8]) -> Result<DataRequestOutput, String> {
+fn deserialize_and_validate_dr_bytes(
```

Comment:
should we have a test for this function?

Reply:
I think we should wait until the error codes are well defined, otherwise we will be rewriting all the tests. Then we can simply test if the error converted to cbor matches the expected error.

Reply:
PR #1909 is already included

Reply:
Added a few tests